docs/sql/float.md

```diff
@@ -89,14 +89,50 @@ REAL '1.23'  -- string style
   </tr>
   <tr>
     <td><code>SIN(value)</code></td>
-    <td>The sine of value as radians. <code>sin</code> only supports argument of type double, so all other types are cast to double. Returns a double.</td>
+    <td>The sine of value, returned in radians. <code>sin</code> only supports argument of type double, so all other types are cast to double. Returns a double.</td>
```

Comment:
here it actually was correct, the argument value is in radians, but the result is a number.